nipy/algorithms/registration/groupwise_registration.py

```diff
@@ -86,6 +86,20 @@ def guess_slice_axis_and_direction(slice_info, affine):
         slice_direction = int(slice_info[1])
     return slice_axis, slice_direction
 
+def tr_from_header(images):
+    if isinstance(images, list):
+        no_init_value = True
+        for image in images:
+            tr = image.header.get_zooms()[-1]
+            if no_init_value:
+                images_tr = tr
+                no_init_value = False
+            if tr != images_tr:
+                raise ValueError('TR inconsistent between images.')
+        else:
+            return images_tr
```

Comment:
I went for `no_init_value` instead of the more elegant 

```
if images_tr is None:
            images_tr = tr
```

BEcause I was worried that the logic would be unable to pick up different-TR images if the first n of the list had `image.header.get_zooms()[3] == None` but it seems `.get_zooms()` can never produce `None`. Can you confirm this?